### DISCUSSION TITLE
[Feature Request] Some features that I would like to be added/changed

### DISCUSSION THREAD

**Discussion by ScottsplaysSus** [Tyrrrz/YoutubeDownloader#258] [Q&A]
Hey there, I have some small requests that I hope it can be added to the downloader :)

I would like the devs or contributors to add/change the features below:

1. The 'Inject media tags' option. Please re-add the toggle button back, I usually wouldn't use the feature because there are problems running files with tags(not the problem with this app) in my other devices. I can still remove the tags through third-party apps, but it would be the best if this app has it re-added(just like older versions).
Edit: Thanks for re-adding the feature in the newest version(1.9.1) :D 

2. Sponsorblock integration. It's just a nice feature to have, to save users time by automatically removing sponsors, self-promotion, and a lot of other sections that are unrelated to the main video. I've heard that yt-dlp is supporting it too :D

3. Options to choose bitrate(kb) for MP3 files. Personally, I don't need this feature, but I've seen it getting mentioned in other issue posts, and I hope this feature can be added in the future.

Overall, this is an amazing Youtube downloader, and I really enjoy using it. Keep up the good work :D

**Comment by clicky6** [comment]
Sponsorblock integration & bitrate selection would be very much helpful indeed...
